device/arista/x86_64-arista_7050cx3_32s/Arista-7050CX3-32S-C32/sai.profile

```diff
@@ -1,2 +1,2 @@
-SAI_INIT_CONFIG_FILE=/usr/share/sonic/hwsku/td3-a7050cx3-32s-32x100G.config.bcm
```

Comment:
Why do we need change this path?

Reply:
those behavior defined in https://github.com/Azure/sonic-buildimage/blob/master/platform/broadcom/docker-syncd-brcm/start.sh